src/index.js

```diff
@@ -51,63 +51,64 @@ class HCaptcha extends React.Component {
         this.onloadScript();
       }
     }
-  
+
     componentWillUnmount() { //If captcha gets removed for timeout or error check to make sure iframe is also removed - hCaptcha
-        if(typeof hcaptcha === 'undefined') return 
+        if(typeof hcaptcha === 'undefined') return
         if (this._removed === false) this.removeFrame()
     }
-  
+
     onsubmitCaptcha (event) {
       if (typeof hcaptcha === 'undefined') return
-      
+
       let token = hcaptcha.getResponse(this._id) //Get response token from hCaptcha widget - hCaptcha
+      hcaptcha.reset(this._id) // Reset response to prevent storing

```

Comment:
Hm wonder if this should be after the `onVerify` instead?

Reply:
@gaieges After some testing, we moved it to a manual method that should be triggered. See demo page.

Reply:
ah ok, sounds good too